tests/configlet/util/configlet.py

```diff
@@ -68,7 +68,8 @@ def get_port_channel():
                 "admin_status": "up",
                 "min_links": "1",
                 "mtu": "9100",
-                "members": list(sonic_local_ports)
+                "tpid": "0x8100",
+                "lacp_key": "auto"
```

Comment:
These changes are already captured in #7872

Reply:
@neethajohn Sure. There are more change need to be updated. Such as removing @member and increase QUEUE length.